### DISCUSSION TITLE
Wrong logo featured in the website

### DISCUSSION THREAD

**Discussion by krisu5** [ente-io/ente#2387] [General]
Hi!

I'm maintainer of [aegis-icons](https://aegis-icons.github.io/) and I noticed that you have incorrectly added my project's logo to "Bulk import from other 2FA apps" section:

![screenshot](https://github.com/ente-io/ente/assets/3540275/b85dc659-3f85-4c3e-8ba4-808ce9167764)

I assume you wanted ["Aegis Authenticator"](https://getaegis.app/) ([GitHub](https://github.com/beemdevelopment/aegis)) logo to that section.

[Here's their actual logo / icon](https://github.com/beemdevelopment/Aegis/blob/master/metadata/en-US/images/icon.png).

Yes, my icon project is made for Aegis app, but I'm not associated with Aegis dev team or their app.

**Comment by vishnukvmd** [comment]
Thanks for letting us know, fixed!

**Reply by krisu5** [reply]
Thanks :)
